README.md

```diff
@@ -6,22 +6,84 @@
 ![ES Version](https://img.shields.io/badge/ES-2017-yellow)
 ![Node Version](https://img.shields.io/badge/node-12.x-green)
 
-This is a Javascript library that implements BLS (Boneh-Lynn-Shacham) signatures and supports signature aggregation.
+Javascript library for BLS (Boneh-Lynn-Shacham) signatures and signature aggregation.
+
+## Usage
```

Comment:
Add comment that users wishing to use native bindings must install @chainsafe/blst@0.1.x